packages/wonder-blocks-core/util/types.js

```diff
@@ -93,8 +91,7 @@ type IdRefList = IdRef | Array<IdRef>;
 
 // Source: https://www.w3.org/WAI/PF/aria-1.1/states_and_properties
 // TODO(kevinb): convert to disjoint union of exact object types keyed on role
-// eslint-disable-next-line flowtype/require-exact-type
-export type AriaProps = {
+export type AriaProps = {|
```

Comment:
I didn't think `flow` would allow mixing exact objects with inexact types using the intersection operator.